pkg/kudoctl/resources/dependencies/resolve.go

```
@@ -66,7 +72,9 @@ func Resolve(operatorVersion *kudoapi.OperatorVersion, resolver pkgresolver.Reso
 		edges: []map[int]struct{}{{}},
 	}
 
-	if err := dependencyWalk(&dependencies, &g, root, 0, resolver); err != nil {
+	operatorDir, _ := operatorAbsPath(operatorArgument)
```

Nit: Let's add a comment why it's okay to suppress the error here.